ui/src/containers/Dashboard.svelte

```diff
@@ -2,10 +2,10 @@
   import { userApi, userSession$ } from "../stores";
   import { onDestroy } from "svelte";
   import marked from "marked";
-  import Icon from "../components/Icon.svelte";
+  import Icon from "../components/miscComponents/Icon.svelte";
   import firebase from "firebase/app";
-  import BuildList from "../components/BuildList.svelte";
-  import LoadingFlat from "../components/LoadingFlat.svelte";
+  import BuildList from "../components/buildListCardComponent/BuildList.svelte";
```

Comment:
The folder names should be consistent. 
Change
From 
    buildListCardComponent
To
    buildListCardComponents

Reply:
Done - renamed to buildlistcardcomponents